src/com/oltpbenchmark/api/Worker.java

```diff
@@ -420,7 +439,10 @@ protected final TransactionType doWork(boolean measure, SubmittedProcedure piece
                     // }
 
                     status = TransactionStatus.UNKNOWN;
+
+                    startOperation.setValue(System.nanoTime());
                     status = this.executeWork(conn, next);
```

Comment:
what if we get a retry status?

Reply:
i do think its useful to isolate measurement to successful operations in terms of our own understanding, as you did here, but we should also be measuring the latency a user would experience for said operation, from op start, through each retry, to final completion, when grading ourselves

Reply:
But 1% of the transactions have failures by design.
Should we include those?

Reply:
id say lets match the previous behavior for those?

Reply:
Done.
I have changed the behaviour a bit and showed the actual latency by default and the whole operation latency along with the connection acquisition latency protected by the flag.